types/stream-chat/index.d.ts

```diff
@@ -55,9 +55,9 @@ export interface Event<T = string> {
   type: T;
   message?: MessageResponse;
   reaction?: ReactionResponse;
-  channel?: Channel;
-  member?: User;
-  user?: User;
+  channel?: ChannelResponse;
+  member?: UserResponse;
```

Comment:
this should be `member?: ChannelMemberResponse`

Reply:
Even more accurate 🙂 Change pushed